src/sentry/static/sentry/app/views/organizationDiscover/savedQueries/index.jsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+// import SentryTypes from 'app/sentryTypes';
+
+import LoadingIndicator from 'app/components/loadingIndicator';
+import {t} from 'app/locale';
+
+import {SidebarHeader, SidebarTitle, SidebarToggle, Fieldset} from '../styles';
+
+export default class SavedQueries extends React.Component {
+  static propTypes = {
+    // organization: SentryTypes.Organization.isRequired,
+    // queryBuilder: PropTypes.object.isRequired,
```

Comment:
Should these be here?

Reply:
I've commented these out for now to pass linting, but left them in since we'll need these props for this component eventually (sorry this is quite WIP at the moment)